kie-server-parent/kie-server-api/src/main/java/org/kie/server/api/marshalling/MarshallingFormat.java

```diff
@@ -42,14 +42,14 @@ public static MarshallingFormat fromId( int id ) {
     }
 
     public static MarshallingFormat fromType( String type ) {
-        if ("xstream".equalsIgnoreCase( type )) {
+        if ("xstream".equalsIgnoreCase( type ) || "application/xstream".equalsIgnoreCase( type ) ) {
             return XSTREAM;
-        } else if ("xml".equalsIgnoreCase( type )) {
+        } else if ("xml".equalsIgnoreCase( type ) || "application/xml".equalsIgnoreCase( type ) ) {
```

Comment:
Maybe use `MediaType.APPLICATION_XML` and `MediaType.APPLICATION_JSON`? Not necessary though, just cleaner.